hazelcast-integration/springboot-caching/.gitignore

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
I know that but it seemed redundant to me. Does this project create any file that should be ignored and is not ignored by root `.gitignore`? If this is the case you can put only those files in `.gitignore`.